en/micro-integrator/docs/references/config-catalog.md

```diff
@@ -4159,6 +4164,111 @@ sender.ssl_profile.read_interval = "30s"</code></pre>
                                         <p>The password of the keystore file that is used as the trust store. By default, the product&#39;s trust store is configured for this purpose.</p>
                                     </div>
                                 </div>
+                            </div><div class="param">
+                                <div class="param-name">
+                                  <span class="param-name-wrap"> <code>enable_message_size_validation</code> </span>
+                                </div>
+                                <div class="param-info">
+                                    <div>
+                                        <p>
+                                            <span class="param-type string"> boolean </span>
+                                            
+                                        </p>
+                                        <div class="param-default">
+                                            <span class="param-default-value">Default: <code>false</code></span>
+                                        </div>
+                                        <div class="param-possible">
+                                            <span class="param-possible-values">Possible Values: <code>&quot;true&quot; or &quot;false&quot;</code></span>
+                                        </div>
+                                    </div>
+                                    <div class="param-description">
+                                        <p>If this property is enabled and the payload exceeds the size of the valid payload size, the Micro Integrator will discontinue reading the input stream. This will prevent out-of-memory issues.</p>
```

Comment:
```suggestion
                                        <p>If this property is enabled and the payload exceeds the size of the valid payload size specified by max_message_size_bytes , the Micro Integrator will discontinue reading the input stream. This will prevent out-of-memory issues.</p>
```